src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -741,6 +792,24 @@ public class foo<T>
         }
 
         [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task InvokeWithOpenAngleCommitSeeOnSpace()
+        {
+            var markupBeforeCommit = @"class c
+{
+        /// <$$
+        void foo() { }
+}";
+
+            var expectedCodeAfterCommit = @"class c
+{
+        /// <see$$/>
+        void foo() { }
+}";
+
+            await VerifyCustomCommitProviderAsync(markupBeforeCommit, "see", expectedCodeAfterCommit, commitChar: ' ');
```

Comment:
I'm having a hard time getting this to work. It seems that the `SPACE` character doesn't commit the completion item in the test harness, despite it working perfectly in VS.

```
state.SendTypeChars("see ")
Await state.AssertNoCompletionSession()
```

The assert fails because the completion session is still present. Everything works correctly for other commit characters such as `>`. Any suggestions on what I might need to get this working?